src/main/java/com/me4502/modularframework/module/Module.java

```diff
@@ -34,12 +34,30 @@
 public @interface Module {
 
     /**
+     * The id of this module.
+     * @return The id of this module
+     */
+    String moduleId();
+
+    /**
      * The name of this module.
      * @return The name of this module
      */
     String moduleName();
```

Comment:
I think it may be worth making this default to ID. I'd prefer to not have it require multiple things like this.